NEWS.md

```diff
@@ -1,3 +1,9 @@
+## 19.3.3 2020-09-02
+
+* Upgrade to RMB 30.2.6 (MODINVSTOR-562)
```

Comment:
```suggestion
* Upgrade to RMB 30.2.7 (MODINVSTOR-567)
```

Reply:
Yes, thanks!

Reply:
I do not push the version tag when opening the release pull request but after the release pull request has been approved and merged.